src/widgets/sliderwidget.cpp

```diff
@@ -301,6 +301,8 @@ void Amarok::VolumeSlider::paintEvent(QPaintEvent*) {
   const int padding = 7;
   const int offset = int(double((width() - 2 * padding) * value()) / maximum());
 
+  m_pixmapInset = volumePixmapDraw();
```

Comment:
Do you really need to recompute it for every paint event?
m_pixmapInset is already set in the constructor, and I don't expect it to change. If I'm right, the only reason for this to happen is in case the theme changes, am I right?
If so, it would be better to avoid calling the function everytime (e.g. detect if the theme changed first? Or just don't consider this corner case).
In the past, we had some performance troubles because we were doing too much work on paintEvent functions.
That being said, maybe the drawing is simple enough so it wouldn't impact the performance (worth to check).